src/components/views/game/game/scoreannouncing/AnnouncementModal.tsx

```diff
@@ -154,9 +154,11 @@ export const AnnouncementModal = observer(() => {
                   title={
                     !active
                       ? "It's not your turn to announce yet!"
+                      : !illegalNumber
+                      ? `Here you can choose how many tricks you think you will win!`
                       : `Here you can choose how many tricks you think you will win! As per the rules you are not 
-                      allowed to choose ${illegalNumber} as the total of announced tricks would equal the amount of 
-                      cards!`
+                      allowed to choose ${illegalNumber} as the total of announced tricks would equal the amount 
+                      of cards!`
```

Comment:
ernsthaft 2 verschachtelte terniary statements? bitte nicht

Reply:
But we do have 3 conditions: Inactive, unrestricted and restricted and for the unrestricted it is not correct to state, that the total of announced tricks shall not equal the amount of cards! If you are the second of 2 players you are free to announce every trick. Only if you are the last player you are possibly restricted in what you can choose.

We can also omit this level of information but in the end it was introduced to show this information.
In case we find a working way to bind the tooltip to the buttons and not the whole component this would change, but until then it's like that.

Reply:
> i am not against the conditionals, just against 2 levels of terniary statements.

got it, at least I think....